apps/system/js/nfc_handover_manager.js

```diff
@@ -161,6 +161,11 @@
       this.bluetoothStatusSaved = false;
       this.bluetoothAutoEnabled = false;
 
+      if (typeof(window.navigator.mozBluetooth.onattributechanged) !==
+        'undefined') {
+        this._debug('Bluetooth APIv2 does not support NFC yet');
+        return;
```

Comment:
Previously we initiate nfc_handover_manager in bluetooth_core but after nfc_core refactor the module has been moved out.

Since nfc_handover_manager handle its own bt adapter and the syntax is not compatible with v2. , To make BTv2 work again it must be turned off for now.